payload-test.json

```diff
@@ -0,0 +1,146 @@
+{
+    "name": "ami_test_cluster",
+    "releaseLabel": "emr-5.28.0",
+    "serviceRole": "emr_service_role",
+    "jobFlowRole": "emr_hbase_ingest",
```

Comment:
This should be the role we created in a recent ticket, i.e. the quite open IAM role